algorithms/Similarity/nearest_neighbors/approximate_nearest_neighbors/tg_ann_ExprFunctions.hpp

```diff
@@ -34,7 +34,7 @@ namespace UDIMPL {
   }
 
   template <typename T>
-  inline double tg_similarity_accum(ListAccum<T>& A, ListAccum<T>& B, const string& similarity_type) {
+  inline double tg_similarity_accum(ListAccum<T> A, ListAccum<T> B, const string& similarity_type) {
```

Comment:
Why don't use reference here?

Reply:
This UDF was continuously failing when I passed the ListAccum directly from the GSQL as ref. It does work when I pass them internally as references (from UDF to UDF). Not sure if we can diagnose what the issue is, because I would prefer ref.

Reply:
I think this is ready to be merged, since I know that this is at least functional. Copying by value is not ideal, but functional is better than non-functional..